package.json

```diff
@@ -51,11 +50,9 @@
     "css-loader": "^3.4.2",
     "d3": "^5.15.0",
     "d3-geo": "^1.11.9",
-    "d3-svg-legend": "^2.25.6",
     "express": "^4.16.4",
     "express-session": "^1.15.1",
     "history": "^4.9.0",
-    "leaflet": "^1.6.0",
```

Comment:
I don't want to remove leaflet -- my next goal after tooltips will be to implement the base layers and overlays, which I'll need leaflet for.

Reply:
I included it in my highlights below, but removing `react-d3-library` and `react-leaflet-d3-svg-overlay` is fine by me.

Reply:
Works for me